src/Hl7.Fhir.ElementModel.Tests/ElementNodeTests.cs

```diff
@@ -92,6 +95,23 @@ public void TestFpNavigate()
             Assert.AreEqual("Observation", contained.FirstOrDefault().InstanceType);
         }
 
+        [TestMethod]
+        public void TestFpNavigateCustomResource()
+        {
+            var provider = new StructureDefinitionSummaryProvider(new CustomResourceResolver());
```

Comment:
You could also use the `SnapshotSource` ResourceResolver instead of an own CustomResourceResolver:

```c#
  var resolver  = new CachedResolver(
                                 new SnapshotSource(
                                      new MultiResolver(
                                          ZipSource.CreateValidationSource(), 
                                          new DirectorySource("TestData/TestSd")
                                       ), SnapshotGeneratorSettings.CreateDefault())
                            );
```